Sources/Benchmark/BenchmarkCommand.swift

```diff
@@ -36,6 +36,9 @@ internal struct BenchmarkCommand: ParsableCommand {
         help: "Maximum number of iterations to run when automatically detecting number iterations.")
     var maxIterations: Int?
 
+    @Option(help: "Number of warm-up iterations to run.")
+    var benchmarkFormat: BenchmarkFormat?
```

Comment:
Why not just `format` here? `benchmark` is implied out of the tool/framework context.

Reply:
Also please update the "help:" string above.

Reply:
> Why not just use `format` here? `benchmark` is implied out of the tool/framework context.

I chose this for parity with [google/benchmark](https://github.com/google/benchmark), which supports a `--benchmark_format` option. That said, I'd be just as happy to support `--format`. In general, I think we could remove the word "Benchmark" from a lot of the API where the context is implied.

Reply:
I am still not sure on whether we should for 1-to-1 compatibility here. As a user I would greatly prefer typing a shorter name.